.baseline/checkstyle/checkstyle.xml

```diff
@@ -361,6 +366,10 @@
             <property name="format" value="\bIOUtils\.toString\("/>
             <property name="message" value="Prefer Guava''s [CharStreams,Files,Resources].toString to avoid charset/stream closing issues."/>
         </module>
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="\bIOUtils\.copy\("/>
+            <property name="message" value="Prefer Guava''s ByteStreams.copy to avoid charset/stream closing issues."/>
```

Comment:
👍 . Thanks for adding this rule and the nice explanation.

Reply:
nit: contains 2x '' instead of a single '

Reply:
`''` is used in the checkstyle file to escape `'`